cli/src/commands/init.js

```diff
@@ -69,22 +69,25 @@ const writeGitignore = (gitignoreFile, sections) => {
     fs.writeFileSync(gitignoreFile, gitignore.stringify(sections, format))
 }
 
-const handler = async ({ force, name, cwd, lib }) => {
+const handler = async ({ force, name, cwd, lib, typeScript }) => {
+    // create the folder where the template will be generated
     cwd = cwd || process.cwd()
     cwd = path.join(cwd, name)
     fs.mkdirpSync(cwd)
-    const paths = makePaths(cwd)
+    const paths = makePaths(cwd, { typeScript })
```

Comment:
[Random brainstorming that I think is out of scope for this PR:] It would be cool if passing the `ts` flag here wasn't necessary, but I think that would need a refactoring of the paths to use something like like globs, which may end up being more complex